### PR TITLE
Fixed wrong permission labels for trackers in the "Roles and permissions" edit page.  

### PR DESCRIPTION
**Pull request by jgonzalezdr**

### CODE REVIEW

**Comment by jijeshmohan** [comment]
Hi,
Could you please check this again with latest plugin version(1.0.9) ? I am able to see the labels properly ( name instead of id)

**Comment by jgonzalezdr** [comment]
It does not work every time because when you patch the roles controller, you only patch the actions "show" and "permissions", but you don't patch other actions like "edit", so when you try to edit the permissions given to a role (without executing the actions "show" or "permissions" before), then the proper "translations" are not inserted.

This is an screenshot when I try to edit the permissions of a role just after having restarted redmine:
image

This is an screenshot when showing the permissions summary ("permissions" action), which worked fine:
image

Basically my solution fixes this by removing the individual patches to each action (that you implemented using the alias_method_chain methods) and replacing them with a "before_filter", which is nicer because it makes the "translations" getting inserted before all actions (making the plugin more maintainable and ready for future changes in the roles controller) in a simpler way.

**Comment by jgonzalezdr** [comment]
BTW, I merged your latest changes into my tree to make it easier for you if you decide to incorporate this pull request.

**Comment by jgonzalezdr** [comment]
And fixed the last merge errors...

**Comment by jijeshmohan** [comment]
Thanks @jgonzalezdr , I have merged the pull request
